template/web/src/store/createWagmiConfig.ts

```diff
@@ -13,7 +21,14 @@ export function createWagmiConfig(projectId: string) {
       },
       {
         groupName: 'Other Wallets',
-        wallets: [rainbowWallet, metaMaskWallet],
+        wallets: [
+          rainbowWallet,
+          metaMaskWallet,
+          trustWallet,
```

Comment:
@eragon512 have you tested with these wallets?

Reply:
And for each Wallet, I would like we triple check the HTML is trashed.